src/components/grid.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {StyleSheet, View} from 'react-native';
+
+export const VBox = props => {
+    return <View style={styles.vbox}>{props.children}</View>;
+};
+
+export const VSeparator = props => {
+    if (props.half) {
+        return <View style={styles.vSeparatorHalf}>{props.children}</View>;
+    } else {
+        return <View style={styles.vSeparator}>{props.children}</View>;
+    }
```

Comment:
acho que aqui dava para usar um operador ternário
```suggestion
return props.half ? (<View style={styles.vSeparatorHalf}>{props.children}</View>) : 
                              (<View style={styles.vSeparator}>{props.children}</View>)
```
só da uma conferida se a sintaxe e a formatação está certa